storageFormats/eventMetadata.h

```diff
@@ -11,6 +11,8 @@ class TTree;
 
 
 namespace rpwa {
+	typedef std::map<std::string, std::pair<double, double> > binningMapType;
+	typedef std::map<std::string, double > multibinCenterType;
```

Comment:
I agree with your name suggestions.
We should only agree on a common name for a multi-dimensional bin. Should it be called 'bin', 'multibin', or 'cell'?
In the python interface, it is called multibin, e.g. the corresponding class is called  'multiBin'.
The changes are no problem using Eclips code refactoring tools.